scripts/deploy_preview.sh

```diff
@@ -59,18 +69,23 @@ kubectl create secret docker-registry $SECRET \
   --docker-username=$DOCKER_HUB_USERNAME \
   --docker-password=$DOCKER_HUB_ACCESS_TOKEN -n $NAMESPACE
 
-echo "Add appsmith-ce to helm repo"
-AWS_REGION=ap-south-1 helm repo add $HELMCHART $HELMCHART_URL
+echo "Add appsmith-ee to helm repo"
+AWS_REGION=us-east-2 helm repo add $HELMCHART $HELMCHART_URL
```

Comment:
Do we need the region for `helm repo add`?

```suggestion
helm repo add "$HELMCHART" "$HELMCHART_URL"
```

Reply:
That is for a extra caution. However, i dont think we need it